tests/__init__.py

```diff
@@ -12,7 +12,10 @@
 except ImportError:
     resource = None
 import signal
-import subprocess
+try:
+    import subprocess32 as subprocess  # py2
+except:
```

Comment:
`except:` in Python2 is only allowed together with comment why you need to catch `SystemExit`, `KeyboardInterrupt`, `GeneratorExit `. Otherwise in Python2 it must be `except Exception`. In this particular case, only `except ImportError`.